src/main/scala/com/gu/zuora6for6modifier/Subscription.scala

```diff
@@ -25,26 +25,32 @@ case class Charge(
 object Subscription {
 
   val productRatePlanNamePrefix6For6 = "GW Oct 18 - Six for Six"
-  val productRatePlanNamePrefixMain = "GW Oct 18 - Quarterly"
+  val productRatePlanNamePrefixMain = "GW Oct 18 - "
```

Comment:
I guess old `Guardian Weekly Zone *` are no longer possible going forward?

Reply:
Yes.  These are all new GW subs that have been taken out in the past few months so would expect them to be simple cases.  If any don't match these conditions they'll be thrown out so we can then go through the failures to see why they failed.